logging/runtime/src/main/java/io/quarkiverse/googlecloudservices/logging/runtime/LoggingConfiguration.java

```diff
@@ -0,0 +1,220 @@
+package io.quarkiverse.googlecloudservices.logging.runtime;
+
+import java.util.Map;
+import java.util.Optional;
+
+import com.google.cloud.logging.Severity;
+import com.google.cloud.logging.Synchronicity;
+
+import io.quarkus.runtime.annotations.ConfigGroup;
+import io.quarkus.runtime.annotations.ConfigItem;
+import io.quarkus.runtime.annotations.ConfigPhase;
+import io.quarkus.runtime.annotations.ConfigRoot;
+
+@ConfigRoot(name = "google.cloud.logging", phase = ConfigPhase.RUN_TIME)
+public class LoggingConfiguration {
+
+    /**
+     * Which Google Operations log should be used by default.
+     */
+    @ConfigItem
+    public String defaultLog;
+
+    /**
+     * Enable or disable the Goocle Cloud logging.
+     */
+    @ConfigItem(defaultValue = "true")
+    public boolean enabled;
+
+    /**
+     * Configure base formatting to be either plain text or
+     * structured json.
+     */
+    @ConfigItem(defaultValue = "TEXT")
```

Comment:
I think there is a way to specify the list of allowed values.
Or maybe use an enum

Reply:
I'll have a look.

Reply:
If there is I can't seem to find it. Will supply it in the java docs instead.

Reply:
The way to provide allowed values is via an enum.

Reply:
Sure, and `LogFormat` is an enum. I was hoping `ConfigItem` had a field for allowed values though.